CONTRIBUTORS.md

```diff
@@ -4,7 +4,7 @@
 
 ### Data Team
 - [Example](https://github.com/phil-ociraptor)
-
+- Eva (https://github.com/esu2020) 
```

Comment:
ah - one small nit, I'd like to keep the markdown link format, so it would be:
`- [Eva](https://github.com/esu2020) `

Brackets, no space, then parentheses

Reply:
got it – edited!